### PR TITLE
Added try! example to documentation.md #29234

### PR DESCRIPTION
**Pull request by ghost**
r? @steveklabnik

### CODE REVIEW

**Comment by killercup** [comment]
Thanks, remember to close #29234 after this is merged! :smile:  (Or add `Fix #29234` to your commit message.)

File: src/doc/trpl/documentation.md [hunk 1]
```diff
@@ -373,6 +373,36 @@ we can add the `#[macro_use]` attribute. Second, we’ll need to add our own
 `main()` as well. Finally, a judicious use of `#` to comment out those two
 things, so they don’t show up in the output.
 
+Another case where the use of `#` is handy is when you want to ignore
+error handling. Lets say you want the following,
+
+```rust
```
Comment (steveklabnik) [inline src/doc/trpl/documentation.md]: this should be ignored, so that it doesn't fail

**Comment by steveklabnik** [comment]
This looks good! Yes, if you could fix that failure by marking that first block as ignore, and while you're at it, fix up that commit message, r=me after :)

File: src/doc/trpl/documentation.md [hunk 2]
```diff
@@ -373,6 +373,36 @@ we can add the `#[macro_use]` attribute. Second, we’ll need to add our own
...
+/// ```
+/// use std::io;
+/// # fn f() -> io::Result<()> {
+/// let mut input = String::new(); 
+/// try!(io::stdin().read_line(&mut input));
+/// # Ok(())
+/// # }
+/// # f();
+/// ```
+```
```
Comment (steveklabnik) [inline src/doc/trpl/documentation.md]: Ah! Another issue. So, you actually _need_ something after a documentation comment for this to parse, so add something like

    # fn foo() { }

Does that make sense? That's why the build is failing. I think there's an example in this file...

**Comment by steveklabnik** [comment]
@efindlay sorry about that! you have to actually leave a comment when you push, github won't send a notification :/

**Pull request merged**